Show on Summary, Update Client, Update System (#110)
resim-ai/api-client: CHANGELOG.md

Changes in this section will be included in the next release.

> ### v0.3.8 - Octover 4 2024

Reviewer: I wish it was Oct _over_

Author: 🥁🥁🥁

#### Added

- The ReSim CLI now supports using specific `pool labels` for creating batches, and running test suites and parameter sweeps. Supplying a pool label forces the ReSim Orchestrator to run that batch using an external runner that is compatible with this label combination.
- The ReSim CLI now supports creation of test suites that use the 'show on summary' flag, which enables the latest test suite and its reports to be viewed on an overview dashboard on the ReSim Web App.

> - The ReSim CLI now supports the updating of Systems: making changes to the resource requirements used.

Reviewer: ```suggestion
- The ReSim CLI now supports updating systems by making changes to the resource requirements.
```
